examples/kube-stack-private/variables.tf

```diff
@@ -9,17 +9,17 @@ variable "name" {
 }
 
```

Comment:
I don't see a reason to keep these whitespace updates in this PR.

Reply:
Actually these whitespace updates were done by my tooling's autoformat not me :) If you feel strongly about it I can revert to previous version.